modules/weblogic-admin-only/cloudwatch.tf

```
@@ -1,7 +1,5 @@
 locals {
-  alert_on_errors = toset(compact([
-    "OASYSERR006", // Invalid character in OASYS message
-  ]))
+  alert_on_errors = toset()
```

I think this will error:
```
~ ❯ terraform console
> toset()
╷
│ Error: Not enough function arguments
│
│   on <console-input> line 1:
│   (source code not available)
│
│ Function "toset" expects 1 argument(s). Missing value for "v".
╵

```

You should be able to pass an empty list though:
```suggestion
  alert_on_errors = toset([])
```